tests/test_bot.py

```diff
@@ -8,7 +8,7 @@
 
 class TestBot(object):
     def setup(self):
-        with open('config.yaml', 'r') as f:
+        with open('config.yaml.example', 'r') as f:
```

Comment:
A few tests were failing out-of-the-box since there was no `config.yaml` for them to load.